packages/devcmd-cli/src/index.ts

```diff
@@ -48,15 +48,42 @@ async function isDir(path: string): Promise<boolean> {
   }
 }
 
+async function startProcess(command: string, args: Array<string>, dirPath: string): Promise<number> {
+  const spawnOptions: SpawnOptions = {
+    stdio: 'inherit',
+    cwd: dirPath
+  };
```

Comment:
minor: please put this variable closer to the `spawn` call (I don't see any reason why it should be located out here and captured into the lambda below)

Reply:
Moved the options into the spawn call